src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -42,6 +42,28 @@ private static bool AllCharsInUInt64AreAscii(ulong value)
             return (value & ~0x007F007F_007F007Ful) == 0;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int GetIndexOfFirstNonAsciiByteInLane(Vector128<byte> value, Vector128<byte> bitmask)
```

Comment:
To better communicate that it's an arm64-specific method:
```suggestion
        private static int AdvSimdGetIndexOfFirstNonAsciiByteInLane(Vector128<byte> value, Vector128<byte> bitmask)
```

Reply:
Actually, it looks like existing convention would mandate something like `GetIndexOfFirstNonAsciiByteInLane_Arm64`